src/dom/DomEvent.js

```diff
@@ -263,23 +263,24 @@ export function getMousePosition(e, container) {
 	);
 }
 
+// @function getWheelPxFactor(): Number
+// Gets the wheel pixel factor based on the devicePixelRatio
+export function getWheelPxFactor() {
+	// We need double the scroll pixels (see #7403 and #4538) for all Browsers
+	// except OSX (Mac) -> 3x, Chrome running on Linux 1x
+	return (Browser.linux && Browser.chrome) ? window.devicePixelRatio :
+		Browser.mac ? window.devicePixelRatio * 3 :
+		window.devicePixelRatio > 0 ? 2 * window.devicePixelRatio : 1;
+}

```

Comment:
Does this have to be a function, or could we export a constant instead?

Reply:
I think a constant would be ok too. I didn't know if we want to have constants in `DomEvent` so I created a function.

Thinking loud: What if the user zooms on the page (not map) and changes the `devicePixelRatio`? With a function we would always have the correct value. How much performance costs do we have with a function instead of a constant?

Reply:
Yeah, devicePixelRatio possibly changing is a good point. Let's keep it a function then.